core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -37,6 +37,7 @@ class LogicalPlanGenerator(val expr: ExpressionGenerator, val explicitDistinct:
     case t: ir.TableSample => tableSample(ctx, t)
     case a: ir.AlterTableCommand => alterTable(ctx, a)
     case ir.NoopNode => ""
+    case ir.UnresolvedCommand(inputText) => s"--${inputText}"
```

Comment:
Added this back, as unresolvedCommand Node can be surfaced in between any node traversal?

Only Immediate Children is tackled inside `batch` method.

Reply:
We are handlelling it inside the batch(ctx, b),
`  private def batch(ctx: GeneratorContext, b: Batch): String = {
    val seqSql = b.children
      .map {
        case ir.UnresolvedCommand(text) =>
          s"-- ${text.stripSuffix(";")}"
        case query => s"${generate(ctx, query)}"
      }
      .filter(_.nonEmpty)
    if (seqSql.nonEmpty) {
      seqSql.mkString("", ";\n", ";")
    } else {
      ""
    }`

 As "generate" is a recursive function. We can remove the  `case ir.UnresolvedCommand(inputText) => s"--${inputText}".`